Build.PL

```diff
@@ -22,8 +22,8 @@ my $builder = Alien::Base::ModuleBuild->new(
     alien_isolate_dynamic => 1,
     alien_repository => {
         protocol => 'https',
-        exact_filename => 'https://github.com/edenhill/librdkafka/archive/v0.11.6.tar.gz',
-        exact_version => 'v0.11.6',
+        exact_filename => 'https://github.com/edenhill/librdkafka/archive/v1.1.0.zip',
```

Comment:
Is there a reason to use the .zip here instead of .tar.gz?  The latter is usually more reliable than the former.